package.json

```diff
@@ -4,6 +4,8 @@
   "private": false,
   "dependencies": {
     "@babel/runtime": "7.14.0",
+    "@data-driven-forms/pf4-component-mapper": "^3.6.4",
```

Comment:
Oh also please use exact versions, that way they get picked up by the cockpit npm update bot one by one. So just `3.6.4`.

Reply:
Sure thing!